docker-compose/docker_compose_test.sh

```diff
@@ -13,9 +13,30 @@
 #
 set -eux -o pipefail
 
+cd "$(dirname "${0}")"
 export COMPOSE_FILE='docker-compose.yml'
 export COMPOSE_PROFILES=test-extensions
-cd "$(dirname "${0}")"
+export PARALLEL_COMPUTES=${PARALLEL_COMPUTES:-1}
+READY_MESSAGE="All computes are started"
+COMPUTES=()
+for i in $(seq 1 "${PARALLEL_COMPUTES}"); do
+  COMPUTES+=("compute${i}")
+done
+CURRENT_TMPDIR=$(mktemp -d)
+cp docker-compose.yml docker-compose.yml.bak
+trap 'mv docker-compose.yml.bak docker-compose.yml; rm -rf ${CURRENT_TMPDIR}' EXIT
+if [[ ${PARALLEL_COMPUTES} -gt 1 ]]; then
+  # Replace the environment variable PARALLEL_COMPUTES with the actual value
+  yq eval -i ".services.compute_is_ready.environment |= map(select(. | test(\"^PARALLEL_COMPUTES=\")) | \"PARALLEL_COMPUTES=${PARALLEL_COMPUTES}\") + map(select(. | test(\"^PARALLEL_COMPUTES=\") | not))" docker-compose.yml
+  for i in $(seq 2 "${PARALLEL_COMPUTES}"); do
+    yq eval -i ".services.compute${i} = .services.compute1" docker-compose.yml
+    yq eval -i "(del .services.compute${i}.build) | (del .services.compute${i}.ports) | (del.services.compute${i}.networks)" docker-compose.yml
```

Comment:
Oh, just noticed, I _think_ this last `(del.services.compute${}.networks)` is a typo (missing space after `(del` before `.services`)